recipes/prefect-dask/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "prefect-dask" %}
+{% set version = "0.2.0.post1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a3ef967b8f47b807c1c46745253c2bebcbfa41783a60d62a3b4c640d158a6e01
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - distributed >=2022.2.0
+    - prefect >=2.3.0
+    - python >=3.6
```

According to setup.py: `python_requires=">=3.7"`, see https://github.com/PrefectHQ/prefect-dask/blob/v0.2.0.post1/setup.py#L27

```suggestion
    - python >=3.7
  run:
    - distributed >=2022.2.0
    - prefect >=2.3.0
    - python >=3.7
```